theme/toolbox.less

```diff
@@ -259,6 +259,50 @@ div.blocklyTreeIcon span {
     background-color: #e4e4e4;
 }
 
+/*******************************
+    Inverted Toolbox
+*******************************/
+#root.hc .invertedToolbox div div {
+    div.blocklyTreeRow .blocklyTreeIcon {
+        color: var(--block-meta-color);
+    }
+
+    div.blocklyTreeRow.blocklyTreeSelected .blocklyTreeIcon {
+        color: #fff;
+    }
+}
+
+.invertedToolbox .blocklyTreeRoot div div {
+    div.blocklyTreeRow {
+        background-color: var(--block-meta-color);
+        color: #fff;
+    }
+
+    div.blocklyTreeRow.blocklyTreeSelected {
+        background-color: var(--block-meta-color);
+    }
+
+    div.blocklyTreeRow:hover {
+        background-color: var(--block-faded-color);
+        color: #fff;
+    }
+}
+
+/*******************************
+    Colored Toolbox
+*******************************/
+.coloredToolbox .blocklyTreeRoot div div {
+    div.blocklyTreeRow {
+        color: var(--block-meta-color);
+        border-left: var(--block-category-border);
```

Comment:
I think this will still need rtl handling so that it "looks right"; check arabic language in micro:bit. I manually set border-left on basic category as an example vs. current behavior, you can see it is on the opposite side from the icon: 
![image](https://github.com/microsoft/pxt/assets/5615930/6c17f60b-e079-4517-aed7-bff5cb7145d5)

when in an rtl language we set the rtl class on body root so you can start the selector with `.main.rtl` to set the special handling for it -- e.g.

```
.main.rtl .coloredToolbox .blocklyTreeRoot .blocklyTreeRow {
    border-left: none;
    border-right: 8px solid var(--block-meta-color);
}
```

Reply:
Are we worried about an override of sorts? Even though the code is `border-left`, in the live site, when I choose Arabic, it gets changed to `border-right`.
![image](https://github.com/microsoft/pxt/assets/49178322/34a07589-be2f-4cf7-867b-84f55a92e5f1)

Reply:
Yeah, it's what I'm getting on local serve with my new changes. When I added the special case to handle it, because I also figured we'd need a special case to ensure it was working, the styling was actually messed up. I'll retry this, though and see what happens.

Reply:
This is what it looks like with the specific style: 
<img width="345" alt="image" src="https://github.com/microsoft/pxt/assets/49178322/c6e899c9-fc05-42e1-ac94-340f87172fc2">

![image](https://github.com/microsoft/pxt/assets/49178322/ea3bd916-8b8b-45a8-b78f-3264d41f8f39)

Reply:
I think the -right and -left are reversed in that css rule screenshot? default / non-rtl version should be setting `border-left`, and `.main.rtl` should be `none`ing `border-left` and setting `border-right`

Reply:
ah, okay, chatted online, I didn't realize that our less preprocessor step actually swaps border-left to border-right and vice versa automatically when making `rtlsemantic.css`. Fun / something to remember, nice to know we don't have to edge case things